Engine_Revit_UI/Convert/Common/ToBHoM/Material.cs

```diff
@@ -98,5 +109,44 @@ internal static oM.Common.Materials.Material ToBHoMMaterial(this Material materi
         }
 
         /***************************************************/
+        /****             Private methods               ****/
+        /***************************************************/
+
+        private static void Update(this oM.Common.Materials.Material material_Destination, Material material_Source)
+        {
+            if (material_Source == null)
+                return;
+
+            ElementId aElementId = material_Source.StructuralAssetId;
```

Comment:
Could we add a warning if this happens, to alert users that they are getting a zeroed out material?